src/shiver/models/corrections.py

```diff
@@ -0,0 +1,202 @@
+"""Model for Corrections tab"""
+# pylint: disable=no-name-in-module
+import time
+from typing import Tuple
+from mantid.api import mtd, AlgorithmManager, AlgorithmObserver
+from mantid.kernel import Logger
+
+logger = Logger("SHIVER")
+
+
+class CorrectionsModel:
+    """Corrections table model"""
+
+    def __init__(self) -> None:
+        self.algorithms_observers = set()  # need to add them here so they stay in scope
+        self.error_callback = None
+
+    def apply(
+        self,
+        ws_name: str,
+        detailed_balance: bool,
+        hyspec_polarizer_transmission: bool,
+        temperature: str = "",
+    ) -> None:
+        """Apply corrections"""
+        output_ws_name = f"{ws_name}_correction"
```

Comment:
There should be some indication which corrections were applied. For example, add "_DB" for detailed balance and "_PT" for the Hyspec polarizer transmission"